creator-node/prod.env

```diff
@@ -7,7 +7,6 @@ delegatePrivateKey=
 spOwnerWallet=
 
 # Infra
-dbUrl=postgres://postgres:postgres@db:5432/audius_creator_node
```

Comment:
are we sure this works? doesn't mediorum default to 5434 or something? or is that just dev

Reply:
i looked through mediorum usages of this env var and didn't see any that this would break given that https://github.com/AudiusProject/audius-protocol/pull/6918 changes stage and prod to hardcode instead of using the env var.

some dev entrypoints still use an env var, but it looks like env var for all those gets set from other sources like CI

Reply:
i'm not aware of anything using 5434 tho

Reply:
okay cool might have been something with the mediorum "single" or audius-compose setup